LoopKit/InsulinKit/DoseStore.swift

```diff
@@ -924,6 +1022,62 @@ extension DoseStore {
             completion(.success(reconciledDoses))
         }
     }
+    
+    /// *This method should only be called from within a managed object context block.*
+    ///
+    /// - Parameter startDate: The earliest outside dose event date to include
+    /// - Returns: An array of logged dose managed objects, in reverse-chronological order
+    /// - Throws: An error describing the failure to fetch objects
+    public func getLoggedDoses(since startDate: Date, completion: @escaping (_ result: DoseStoreResult<[PersistedOutsideDose]>) -> Void) {
```

Comment:
[nit] Similar to above, consider `getStoredDoses` (also, fix all the comments in this file to use the verb "store" instead of "log")

Reply:
See refactoring for "manuallyEntered"